readme.txt

```diff
@@ -0,0 +1,31 @@
+
+=== Plugin Check ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.0
+Tested up to:      6.1
+Requires PHP:      5.6
+Stable tag:        0.1.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, testing, security
+
+Plugin Check plugin from the WordPress Performance Team, a collection of tests to help improve plugin performance.
+
+== Description ==
+
+The Plugin Check plugin can be used to run a number of performance and security related tests against plugins and provide useful insights on how they can be improved.
+
+== Installation ==
+t
+= Installation from within WordPress =
+
+1. Visit **Plugins > Add New**.
+2. Search for **Plugin Check**.
+3. Install and activate the Plugin Check plugin.
+
+= Manual installation =
+
+1. Upload the entire `plugin-check` folder to the `/wp-content/plugins/` directory.
+2. Visit **Plugins**.
+3. Activate the Plugin Check plugin.
```

Comment:
This `readme.txt` file looks good for now, but can you open an issue (for some time in the future, not urgent) to come up with a complete readme file? This would be good to log as an issue so we remember it for prior to launching anything.

Reply:
Thanks @felixarntz new issue created here https://github.com/10up/plugin-check/issues/45